src/main/java/software/bernie/techarium/registry/BlockRegistry.java

```diff
@@ -58,9 +61,15 @@
     public static final RegistryObject<TileEntityType<TopEnabledOnlySlave>> ARBORETUM_TOP_TILE = TILES.register("arboretum_top", () -> TileEntityType.Builder.of(TopEnabledOnlySlave::new, ARBORETUM_TOP.get())
             .build(null));
 
+    // Ores
+    public static final BlockRegistryObjectGroup<Block, BlockItem, TileEntity> ALUMINIUM_ORE =
```

Comment:
Can we remove this abstraction and just use RegistryObject<Block> here? This seems like too much